assets/js/modules/analytics/components/settings/GA4DashboardViewToggle.js

```diff
@@ -36,7 +41,7 @@ import { trackEvent } from '../../../../util';
 import useViewContext from '../../../../hooks/useViewContext';
 const { useSelect, useDispatch } = Data;
 
-export default function GA4DashboardViewToggle() {
+export default function GA4DashboardViewToggle( { isUAAvailable = false } ) {
```

Comment:
"Availability" for the module in other contexts has been more analogous to "exists" where as this means something else (connected + enabled).

How about we pass `isUAConnected` and `isUAEnabled` as separate props here instead?

Reply:
SGTM, this is done too.